### PR TITLE
new project -url shortener

### PR DESCRIPTION
**Pull request by PraiseTy**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+MONGO_URI = " "
```
Comment (greatertomi) [inline .env.example:1]:
```suggestion
MONGO_URI=
```

File: db/connect.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const mongoose = require('mongoose')
+
+const connectDB = (url) => {
+    return mongoose.connect(url, { });
+};
```
Comment (greatertomi) [inline db/connect.js:5]: It's nice to have error logs and success message.

```suggestion
const connectDB = async (url) => {
  try {
    await mongoose.connect(url);
    console.log('MongoDB Connected');
  } catch (err) {
    console.error('Error connecting to MongoDB Atlas:', err);
  }
};
```

File: routes/main.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const express = require('express')
+const { getAllUrls, getUrl, editUrl, deleteUrl } = require('../controllers/main')
+
+router = express.Router()
+
+
+router.route('/').get(getAllUrls)
+router.route('/:id').get(getUrl).put(editUrl).delete(deleteUrl)
```
Comment (greatertomi) [inline routes/main.js:8]: Changing HTTP methods like this looks cool, but it's generally a good idea. Split them up

**Review by greatertomi — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (PraiseTy) [reply .env.example]: done on 0188349

File: db/connect.js (hunk 2 above)
Comment (PraiseTy) [reply db/connect.js]: change done on bd42f89

File: routes/main.js (hunk 3 above)
Comment (PraiseTy) [reply routes/main.js]: done on 48ddf7b

**Review by greatertomi — APPROVED** [review]

**Pull request merged**